src/admin/config.yml

```diff
@@ -0,0 +1,39 @@
+backend:
+  name: git-gateway
+  branch: decap-integration
+  commit_messages:
+  create: Create {{collection}} “{{slug}}”
+  update: Update {{collection}} “{{slug}}”
+  delete: Delete {{collection}} “{{slug}}”
+  uploadMedia: Upload “{{path}}”
+  deleteMedia: Delete “{{path}}”
+  openAuthoring: '{{message}}'
+
+
+media_folder: "src/assets/uploads"
+public_folder: "/assets/uploads"
+
+site_url: https://taakconsult.netlify.app/
```

Comment:
This should be https://taakconsulting.net/

Reply:
okay, let me change this